README.md

```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
+tap "homebrew/core"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
-tap "caskroom/fonts"
+tap "homebrew/cask-fonts"
+
+brew "emacs"
 brew "imagemagick"
 brew "mysql@5.6", restart_service: true, link: true, conflicts_with: ["mysql"]
-brew "emacs", args: ["with-cocoa", "with-gnutls"]
```

Comment:
Would be good to use an example from a third-party tap with options e.g. `denji/nginx/nginx-full --with-rtmp-module`

Reply:
```suggestion
brew "denji/nginx/nginx-full", args: ["with-rmtp-module"]
```

Reply:
Looks good.